modules/ROOT/pages/design-add-api-dependency.adoc

```diff
@@ -1,42 +1,36 @@
-= Add a RAML API Fragment to an API-Specification Project as a Dependency
+= Adding Dependencies to an API-Specification Project
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-:imagesdir: ../assets/images
+In projects for API specifications or API fragments, you can add RAML API fragments or
+rulesets as dependencies directly from Anypoint Exchange.
 
-In projects for API specifications or API fragments, you can add RAML API fragments as dependencies directly from Anypoint Exchange.
+The dependencies are listed in your projects,
+but they are not present in your projects. Instead, the text editor references them
```

Comment:
If you use the suggested text from the last comment, you can delete lines 10, 11, and 12.